pages/order-list/index.js

```diff
@@ -1,17 +1,37 @@
 import ShoppingCart from "/components/shopping-cart";
 import Drinks from "/components/order-menu/drink-menu";
 import Maindishes from "/components/order-menu/main-menu";
+import OrderOverview from "../order-overview";
+import Backdrop from "@/components/back-drop/index.js";
 import BackButton from "@/components/back-button";
+import { useState } from "react";
 
 export default function OrderList({ handleAmountChange, items }) {
+  const [drawerOpen, setDrawerOpen] = useState(false);
+
+  const drawerToggleClickHandler = () => {
+    setDrawerOpen(!drawerOpen);
+  };
+
+  const backdropClickHandler = () => {
+    setDrawerOpen(false);
+  };
+
+  let backdrop;
+  if (drawerOpen) {
+    backdrop = <Backdrop close={backdropClickHandler} />;
+  }
+
   return (
     <>
       <h1>Order Menu</h1>
       <h2>Hauptspeisen</h2>
       <Maindishes handleAmountChange={handleAmountChange} items={items} />
       <h2>Getränke</h2>
       <Drinks handleAmountChange={handleAmountChange} items={items} />
-      <ShoppingCart />
+      <OrderOverview items={items} show={drawerOpen} />
+      {backdrop}
```

Comment:
```suggestion
      {drawerOpen && <Backdrop close={backdropClickHandler} />}
```
This is essentially the same thing but more in line with how we learned to toggle the visibility of components.